journal-writings/modernized/15-keeping-negroes.adoc

```diff
@@ -0,0 +1,1774 @@
+
```

Comment:
this is no big deal, but usually I would have no empty line here at the top of a file.